tee/rpc/node.go

```diff
@@ -0,0 +1,127 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package rpc
+
+import (
+	"bytes"
+	"encoding/gob"
+	"fmt"
+	"net"
+	"net/rpc"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/rlp"
+	"github.com/perun-network/erdstall/tee"
+
+	"perun.network/go-perun/log"
+	"perun.network/go-perun/pkg/sync"
+	"perun.network/go-perun/pkg/sync/atomic"
+)
+
+// Node is a server that exposes an Enclave as a slave.
+type Node struct {
```

Comment:
Not sure if `Node` is the best name for this. A node is usually one entity in a network, like an ethereum node. This here is rather a server, where the client is the `RemoteEnclave`. So maybe we call this something like `RPCEnclave`. We should also sync with @ggwpez since he's having similar naming issues in the operator in #61